<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteFlight</class>
 <widget class="QMainWindow" name="deleteFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="beginPlace">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="beginTime">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入终点:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入航班到达时间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endPlace">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>删除该趟航班</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入航班发车时间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endTime">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入起点:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAllButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>删除全部</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showSign">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>320</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
